components/google_drive/sources/new-files-instant/new-files-instant.js

```diff
@@ -111,15 +111,17 @@ module.exports = {
 
     // Component was invoked by timer
     if (event.interval_seconds) {
-      const {
+      let {
         channelID,
         pageToken,
         expiration,
         resourceId,
       } = await this.googleDrive.invokedByTimer(
         this.drive,
         subscription,
-        this.http.endpoint
+        this.http.endpoint,
+        this.db.get("channelID"),
+        this.db.get("pageToken")
```

Comment:
We could reuse the variables declared at the beginning of the method, right?
Just like we do with `subscription`